### PR TITLE
`Development`: Recommend IntelliJ in student exam guide

### PR DESCRIPTION
**Pull request by Strohgelaender**
#### General
- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context
We should update the recommended tools in the exam guide before the start of the new exam period.

### Description
Updated eclipse version to 2022-06 and added recommendation for IntelliJ. 

### Steps for Testing
View the new documentation at https://artemis-platform--5357.org.readthedocs.build/en/5357/user/exams/students_guide/.

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### CODE REVIEW

File: src/main/webapp/i18n/en/guidedTourCourseExerciseOverview.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
+                "content": "Before we have a look at this course and the example exercise, we need to download and install a few software tools.<ul><li>Please download and install the newest <a href='https://www.oracle.com/java/technologies/javase-downloads.html' target='_blank' rel='noopener noreferrer'>Java JDK</a> (at least Java JDK 17). Make sure to choose JDK and <span class='red bold'>not</span> JRE.</li><li>Please download and install <a href='https://www.jetbrains.com/en-en/idea/download/' target='_blank' rel='noopener noreferrer'>IntelliJ</a> (2022.1.3 or later)</li><li>Please download and install <a href='https://www.sourcetreeapp.com' target='_blank' rel='noopener noreferrer'>SourceTree</a> with git (4.1.5 or later for macOS and 3.4.6 or later for Windows).</li></ul>",
```
Comment (Marvin-Vitcu) [inline src/main/webapp/i18n/en/guidedTourCourseExerciseOverview.json:6]: this download link to the English version of IntelliJ does not work. Get 404 error

**Review by babyygemperor — DISMISSED** [review]
`find . -type f -exec cat {} \; | grep 'Eclipse'` didn't fetch any matching result.
LGTM 👍🏼

File: docs/user/exams/students_guide.rst [hunk 2]
```diff
@@ -25,11 +25,11 @@ Prerequisites
 
     - **Java IDE with JDK 17**
 
-        **Recommendation:** Eclipse IDE 2021‑12.
+        **Recommendation:** IntelliJ IDEA 2022.1.3.
```
Comment (dfuchss) [inline docs/user/exams/students_guide.rst:28]: Why do you want to drop the recommendation for eclipse?
I would suggest to simply add the recommendation for intellij here.

File: src/main/webapp/i18n/de/guidedTourCourseExerciseOverview.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
         "courseExerciseOverview": {
             "installPrerequisites": {
                 "headline": "Installiere die notwendigen Tools",
-                "content": "Bevor wir uns den Kurs und die darin enthaltene Beispielaufgabe anschauen, müssen wir zuerst die notwendigen Tools herunterladen und installieren.<ul><li>Installiere das neueste <a href='https://www.oracle.com/java/technologies/javase-downloads.html' target='_blank' rel='noopener noreferrer'>Java JDK</a> (mind. Java JDK 17). Stelle dabei sicher, dass du das JDK und <span class='red bold'>nicht</span> JRE auswählst.</li><li>Installiere die <a href='https://www.eclipse.org/downloads/packages/' target='_blank' rel='noopener noreferrer'>Eclipse IDE for Java Developers</a> (2021-12 oder neuer)</li><li>Installiere die neueste Version von <a href='https://www.sourcetreeapp.com' target='_blank' rel='noopener noreferrer'>SourceTree</a> mit git (4.1.5 oder neuer für macOS bzw. 3.4.6 oder neuer für Windows)</li></ul>",
```
Comment (krusche) [inline src/main/webapp/i18n/de/guidedTourCourseExerciseOverview.json:6]: the guided tour still relies on Eclipse, so please revert this change and let's focus on the student exam guide in this PR

File: src/main/webapp/i18n/en/guidedTourCourseExerciseOverview.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
...
+                "content": "Before we have a look at this course and the example exercise, we need to download and install a few software tools.<ul><li>Please download and install the newest <a href='https://www.oracle.com/java/technologies/javase-downloads.html' target='_blank' rel='noopener noreferrer'>Java JDK</a> (at least Java JDK 17). Make sure to choose JDK and <span class='red bold'>not</span> JRE.</li><li>Please download and install <a href='https://www.jetbrains.com/idea/download/' target='_blank' rel='noopener noreferrer'>IntelliJ</a> (2022.1.3 or later)</li><li>Please download and install <a href='https://www.sourcetreeapp.com' target='_blank' rel='noopener noreferrer'>SourceTree</a> with git (4.1.5 or later for macOS and 3.4.6 or later for Windows).</li></ul>",
```
Comment (krusche) [inline src/main/webapp/i18n/en/guidedTourCourseExerciseOverview.json:6]: the guided tour still relies on Eclipse, so please revert this change and let's focus on the student exam guide in this PR

File: docs/user/exams/students_guide.rst (hunk 2 above)
Comment (krusche) [reply docs/user/exams/students_guide.rst:28]:
```suggestion
        **Recommendation:** IntelliJ IDEA 2022.1.3 or Eclipse IDE 2022-06
```

Comment (krusche) [reply docs/user/exams/students_guide.rst:28]: We completely switched to IntelliJ with Gradle in our course. It's much easier and more reliable to use than Eclipse ;-)

File: docs/user/exams/students_guide.rst [hunk 5]
```diff
@@ -25,11 +25,11 @@ Prerequisites
...
 
     - **Git Client**
 
-        **Recommendation:** SourceTree
+        **Recommendation:** IntelliJ comes with an integrated git client
```
Comment (krusche) [inline docs/user/exams/students_guide.rst:32]:
```suggestion
        **Recommendation:** Use the git client integrated into IntelliJ and Eclipse! Alternative: SourceTree or GitKraken
```

File: docs/user/exams/students_guide.rst [hunk 6]
```diff
@@ -25,11 +25,11 @@ Prerequisites
 
     - **Java IDE with JDK 17**
```
Comment (krusche) [inline docs/user/exams/students_guide.rst:26]: please change line 24 above to:

- The following prerequisites are only required if your exam contains Java programming exercises:

**Review by luisa-we — APPROVED** [review]
Documentation looks good to me!

**Review by Marvin-Vitcu — APPROVED** [review]

**Review by krusche — APPROVED** [review]
